cmd/policy.go

```diff
@@ -138,6 +170,7 @@ func init() {
 	genSubCmd.Flags().StringVar(&prBranch, "prbranch", "", "The branch that will be used for creating the PR - this is the branch that gets pushed to remote")
 	genSubCmd.MarkFlagRequired("prbranch")
 	policyCmd.AddCommand(genSubCmd)
+	policyCmd.AddCommand(terraformSubCmd)
```

Comment:
Whatever name is decided, all function and variable names will need to be updated.

Reply:
Done :)